packages/api/test/unit/platforms/vtex/clients/commerce.test.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  type Options,
+  getContextFactory,
+} from '../../../../../src/platforms/vtex'
+import * as clients from '../../../../../src/platforms/vtex/clients'
+import type { Clients } from '../../../../../src/platforms/vtex/clients'
+
+const apiOptions = {
+  platform: 'vtex',
+  account: 'storeframework',
+  environment: 'vtexcommercestable',
+  channel: '{"salesChannel":"1"}',
+  locale: 'en-US',
+  subDomainPrefix: ['www'],
+  hideUnavailableItems: false,
+  incrementAddress: false,
+  flags: {
+    enableOrderFormSync: true,
+  },
+} as Options
+
+const contextFactory = getContextFactory(apiOptions)
+const context = contextFactory({})
+const getClients = clients.getClients
+
+const pickupPointsMocked = jest.fn()
+
+jest
+  .spyOn(clients, 'getClients')
+  .mockImplementation((options, context): Clients => {
+    const otherClients = getClients(options, context)
+
+    return {
+      ...otherClients,
+      commerce: {
+        ...otherClients.commerce,
+        checkout: {
+          ...otherClients.commerce.checkout,
+          pickupPoints: pickupPointsMocked,
+        },
+      },
+    }
+  })
+
+describe('VTEX Commerce', () => {
+  describe('Checkout', () => {
+    describe('Pickup points', () => {
```

Comment:
question: not sure what those tests are testing, wouldn't be better to mock fetchAPI? 
Because the tests are mocking, calling and checking the same function (commerce.checkout.pickupPoints)